Screens/Authentication/log-in/log-in.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>gozo Store | Log-In</title>
+    <link rel="stylesheet" href="../signup-log-in-card.css">
+</head>
+<body>
+
+    <nav class="navbar bg-yellow">
+    
+        <div class="navbar-header">
+            <a href="#"><p class="navbar-header-title text-l text-gray">gozo Stores</p></a>
+        </div>
+    
+        <div class="navbar-links">
+
+            <span class="searchbar">
+                <span class="input-group">
+                    <input type="text" placeholder="find your product" />
+                </span>
+                <a href="#"><img class="icon" src="../../../assets/icons/search.png" alt="search"></a>
+            </span>
+
+            <div class="icon-with-badge">
+                <a href="#"><img class="icon" src="../../../assets/icons/heart-outline.png" alt="wishlist"></a>
+                <span class="badge bg-lightblue">9</span>
+            </div>
+
+            <div class="icon-with-badge">
+                <a href="#"><img class="icon" src="../../../assets/icons/cart-outline.png" alt="cart"></a>
+                <span class="badge bg-lightblue">5</span>
+            </div>
+
+            <button class="btn btn-lightblue"><a href="#">Log In</a></button>
+        </div>
+    </nav>
+
+    <main>
+        <div class="log-in-container">
+
+            <form action="" method="post">
+                <div class="card shadow">
+
+                    <h3 class="title">Log In</h3>
+                    <div class="input-group">
+                        <label for="email">Email address</label>
+                        <input type="email" name="email" id="" placeholder="admin@gozo.com" required/>
+                    </div>
+                    <div class="input-group">
+                        <label for="password">Password</label>
+                        <input type="password" name="password" id="" required/>
+                    </div>
+                    <div class="login-options">
+                        <input type="checkbox" name="TnC" id="">Remember me <br>
+                        <a href="#">Forgot your Password?</a>
```

Comment:
Suggestion: Please provide an id or remove if not needed

Reply:
> Suggestion: Please provide an id or remove if not needed

okay, I'll take care of it